### ISSUE TITLE
Questions about using this script

### ISSUE THREAD

**Issue by Philipp0205** [telecastr/tp2ctl#4]
Hey there,

first of all thank you for writing the script. I really appreciate that somebody tries to make this keyboard properly usable on Linux.

When i run 
`sudo python3 tp2ctl.py -s 8`
i get the following stacktrace.

```
Traceback (most recent call last):
  File "/home/philipp/Downloads/tp2ctl/src/tp2ctl/tp2ctl.py", line 134, in <module>
    main()
  File "/home/philipp/Downloads/tp2ctl/src/tp2ctl/tp2ctl.py", line 101, in main
    raise RuntimeError('Could not find Lenovo Trackpoint II Keyboard')
RuntimeError: Could not find Lenovo Trackpoint II Keyboad
```

Then I used a script which show me which hidraw devices corresponds to what physical devices 

```
sudo sh hidraw.sh
hidraw0          SYNA8006:00 06CB:CD8B
hidraw1          ITE Tech. Inc. ITE Device(8910)
hidraw2          SteelSeries SteelSeries GameDAC
hidraw3          SteelSeries SteelSeries GameDAC
hidraw4          SteelSeries SteelSeries GameDAC
hidraw5          SteelSeries SteelSeries GameDAC
hidraw6          TrackPoint Keyboard II
hidraw7          Lenovo Bluetooth Mouse
```
so I sued 
```
sudo python3 tp2ctl.py -d /dev/hidraw6 -s 8
Using hid-raw file handle '/dev/hidraw6'
Sending Data ... Done!
```

BUT nothing changed in my trackpoint sensitivity.

Am I doing something wrong here?

**Comment by telecastr** [comment]
Hi Philip!
Your usage seems correct, I'm wondering why the Keyboard can not be detected in the first place ... 
- Could you please post the output of `lsusb` and `ls /sys/bus/hid/devices` ? (We are looking for Vendor/Device 17EF / 60EE)
-  What OS/Version are you using?

**Comment by Philipp0205** [comment]
Hi telecastr, thank you for your help! 

Here is the output of `lsusb`
```
Bus 006 Device 001: ID 1d6b:0ls /sys/bus/hid/devicesls /sys/bus/hid/devices003 Linux Foundation 3.0 root hub
Bus 005 Device 003: ID 048d:8916 Integrated Technology Express, Inc. ITE Device(8910)
Bus 005 Device 002: ID 05e3:0608 Genesys Logic, Inc. Hub
Bus 005 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 005: ID 06cb:00bd Synaptics, Inc. Prometheus MIS Touch Fingerprint Reader
Bus 001 Device 003: ID 1ls /sys/bus/hid/devices3d3:5405 IMC Networks Integrated Camera
Bus 001 Device 004: ID 1038:1280 SteelSeries ApS SteelSeries GameDAC
Bus 001 Device 007: ID 1038:1282 SteelSeries ApS SteelSeries GameDAC
Bus 001 Device 002: ID 0451:2036 Texas Instruments, Inc. TUSB2036 Hub
Bus 001 Device 006: ID 8087:0026 Intel Corp. 
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
```
and `ls /sys/bus/hid/devices`

```
0003:048D:8916.0002  0003:1038:1280.0004  0003:1038:1282.0006  0005:17EF:60EF.000D
0003:1038:1280.0003  0003:1038:1280.0005  0005:17EF:60E1.000A  0018:06CB:CD8B.0001
```

**Comment by telecastr** [comment]
Is there a possibility that your keyboard is connected via Bluetooth still? (This script is thus far only tested/working with the USB-Dongle.)

The output of `ls /sys/bus/hid/devices` shows a `*:17EF:60EF:*` device, which is the Bluetooth ProductID of this keyboard. Also, lsusb doesn't show the Lenovo Keyboard....

**Comment by Philipp0205** [comment]
Oh yes that was the issue. Thanks!! I may have overread that in the readme. 

A totally unrelated question: Did you manage to swap the Ctrl and the Fn key on the keyboard?

**Comment by telecastr** [comment]
Glad it works!
Please note that the fixes to the Linux Kernel mentioned in #3  also include Bluetooth support 
PS:  I haven't tried to switch any keys ...

**Comment by Philipp0205** [comment]
That's nice! Until my distribution get's that kernel update I will happily use your script now :)
